notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -889,3 +889,51 @@ anti-gravity technology [The HVEC flying vehicle was said to have anti-gravity e
 coping with the death of a spouse [The tragic death of Bernie Hayes' wife shaped him to become a raving conspiracy theorist.]
 
 
+movie: Dune (2021)
+==================
+
+:: Title
+Dune
+
+:: Description
+Dune (titled onscreen as Dune: Part One) is a 2021 American epic science
+fiction film directed by Denis Villeneuve and written by Villeneuve, Jon
+Spaihts, and Eric Roth. It is the first of a planned two-part adaptation of
+the 1965 novel by Frank Herbert, primarily covering the first half of the
+book.
+
+Synopsis: Set in the far future, it follows Paul Atreides as his family, the
```

Comment:
This is a break from the usual synopsis style

Reply:
Good catch. I revised it.